server/offenceCodeDecisions/Decision.ts

```diff
@@ -177,7 +177,15 @@ export default class Decision {
   }
 }
 
+export function decision(question: Question | string) {
+  return new Decision(question)
+}
+
 // eslint-disable-next-line no-shadow
-enum Page {
+export enum DecisionType {
   DEFAULT = 'DEFAULT',
```

Comment:
Is NONE more meaningful - does that indicate there is no input required - or maybe RADIO_SELECTION_ONLY or something?